_data/Tools/tests/test_api.py

```diff
@@ -0,0 +1,19 @@
ingdon', '113.750', ('N051.43.34.000', 'W000.32.59.000')], 'BCN': ['Brecon', '117.450', ('N051.43.32.000', 'W003.15.47.000')], 'BPK': ['Brookmans Park', '117.500', ('N051.44.59.000', 'W000.06.24.000')], 'CLN': ['Clacton', '114.550', ('N051.50.55.000', 'E001.08.51.000')], 'CPT': ['Compton', '114.350', ('N051.29.30.000', 'W001.13.11.000')], 'DTY': ['Daventry', '116.400', ('N052.10.49.000', 'W001.06.50.000')], 'DCS': ['Dean Cross (DME)', '115.200', ('N054.43.19.000', 'W003.20.26.000')], 'DET': ['Detling', '117.300', ('N051.18.14.000', 'E000.35.50.000')], 'DVR': ['Dover (DME)', '114.950', ('N051.09.45.000', 'E001.21.33.000')], 'DUD': ['Dundonald (DME)', '115.450', ('N055.33.32.000', 'W004.36.06.000')], 'GAM': ['Gamston (DME)', '112.800', ('N053.16.53.000', 'W000.56.50.000')], 'GOW': ['Glasgow (DME)', '115.400', ('N055.52.14.000', 'W004.26.45.000')], 'GWC': ['Goodwood (DME)', '114.750', ('N050.51.19.000', 'W000.45.24.000')], 'GLO': ['Green Lowther (DME)', '109.650', ('N055.23.24.000', 'W003.44.12.000')], 'HON': ['Honiley', '113.650', ('N052.21.24.000', 'W001.39.49.000')], 'IOM': ['Isle Of Man', '112.200', ('N054.04.01.000', 'W004.45.49.000')], 'JSY': ['Jersey', '112.200', ('N049.13.16.000', 'W002.02.46.000')], 'LAM': ['Lambourne', '115.600', ('N051.38.46.000', 'E000.09.06.000')], 'LND': ['Lands End', '114.200', ('N050.08.11.000', 'W005.38.13.000')], 'LON': ['London', '113.600', ('N051.29.14.000', 'W000.28.00.000')], 'LYD': ['Lydd (DME)', '114.050', ('N050.59.59.000', 'E000.52.43.000')], 'MAC': ['Machrihanish (DME)', '116.000', ('N055.25.48.000', 'W005.39.01.000')], 'MCT': ['Manchester', '113.550', ('N053.21.25.000', 'W002.15.44.000')], 'MAY': ['Mayfield', '117.900', ('N051.01.02.000', 'E000.06.58.000')], 'MID': ['Midhurst', '114.000', ('N051.03.14.000', 'W000.37.30.000')], 'NEW': ['Newcastle (DME)', '114.250', ('N055.02.18.000', 'W001.41.54.000')], 'OCK': ['Ockham (DME)', '115.300', ('N051.18.18.000', 'W000.26.50.000')], 'OTR': ['Ottringham', '113.900', ('N053.41.54.000', 'W000.06.13.000')], 'PTH': ['Perth', '110.400', ('N056.26.33.000', 'W003.22.07.000')], 'POL': ['Pole Hill', '112.100', ('N053.44.38.000', 'W002.06.12.000')], 'SAB': ['Saint Abbs', '112.500', ('N055.54.27.000', 'W002.12.23.000')], 'SFD': ['Seaford', '117.000', ('N050.45.38.000', 'E000.07.19.000')], 'SAM': ['Southampton (DME)', '113.350', ('N050.57.19.000', 'W001.20.42.000')], 'STN': ['Stornoway', '115.100', ('N058.12.25.000', 'W006.10.59.000')], 'STU': ['Strumble', '113.100', ('N051.59.41.000', 'W005.02.25.000')], 'SUM': ['Sumburgh', '117.350', ('N059.52.44.000', 'W001.17.12.000')], 'TLA': ['Talla', '113.800', ('N055.29.57.000', 'W003.21.10.000')], 'TIR': ['Tiree', '117.700', ('N056.29.36.000', 'W006.52.32.000')], 'TNT': ['Trent', '115.700', ('N053.03.14.000', 'W001.40.12.000')], 'TRN': ['Turnberry', '117.500', ('N055.18.48.000', 'W004.47.02.000')], 'WAL': ['Wallasey', '114.100', ('N053.23.31.000', 'W003.08.04.000')], 'WIK': ['Wick', '113.600', ('N058.27.32.000', 'W003.06.01.000')]} )
```

Comment:
Forgive me if I'm not understanding, but doesn't having all the data in the test defeat the point of the parser? Would the test fail if any data changed?

Reply:
It's tested on a past AIP version (2001) so shouldn't change unless NATS mess up :)